modules/op-scoping-github-token.adoc

```diff
@@ -0,0 +1,110 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="scoping-github-token_{context}"]
+= Scoping the GitHub token to additional repositories
+
+{pac} uses the GitHub App to generate a GitHub access token. {pac} uses this token to retrieve the pipeline payload from the repository and to enable the CI/CD processes to interact with GitHub repositories.
+
+By default, this token is scoped only to the repository from which {pac} retrieves the pipeline definition. In some cases, you might want the token to have access to additional repositories. For example, there might be a CI repository where the `.tekton/pr.yaml` file and source payload might be located, but the build process defined in `pr.yaml` might fetch tasks from a separate private CD repository.
+
+You can extend the scope of the GitHub token in two ways:
+
+* _Global configuration_: You can extend the GitHub token to a list of repositories in different namespaces. You must have administrative permissions to set this configuration.
+* _Repository level configuration_: You can extend the GitHub token to a list of repositories that exist in the same namespace as the original repository. You do not require administrative permissions to set this configuration.
```

Comment:
```suggestion
* _Repository level configuration_: You can extend the GitHub token to a list of repositories that exist in the same namespace as the original repository. You do not need administrative permissions to set this configuration.
```
I'd say either "you don't need" or "are not required" rather than "you do not require", again just a suggestion for readability though